### PR TITLE
Add pyexcel - pyexcel-xls - pyexcel-xlsx - pyexcel-ezodf - pyexcel-od…

### PR DESCRIPTION
**Pull request by williamjamir**
This commit adds the following recipes:
pyexcel 
pyexcel-xls
pyexcel-xlsx 
pyexcel-ezodf 
pyexcel-ods3 
pyexcel-webio 
pyexcel-io

### CODE REVIEW

**Comment by williamjamir** [comment]
Could you give a look at this PR? @synapticarbors

File: recipes/pyexcel-ods3/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/pyexcel-ods3/meta.yaml]: Are the other packages beyond `python` and `setuptools` required to build this package? With `--single-version-externally-managed` you typically only need what `setup.py` imports (directly or indirectly), unless there is a `setup_requires` section.

File: recipes/pyexcel-webio/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyexcel >=0.5.1
```
Comment (synapticarbors) [inline recipes/pyexcel-webio/meta.yaml]: Is `pyexcel` an actual `build` requirement?

File: recipes/pyexcel-xls/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/pyexcel-xls/meta.yaml]: Check necessity of `build` requirements beyond `python` and `setuptools`

File: recipes/pyexcel-xlsx/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/pyexcel-xlsx/meta.yaml]: Check necessity of `build` requirements other than `python` and `setuptools`.

File: recipes/pyexcel-webio/meta.yaml (hunk 2 above)
Comment (williamjamir) [reply recipes/pyexcel-webio/meta.yaml]: It is not necessary any extra package, you are correct. I was basing my recipe on the conda skeleton and looking at setup.py I noticed that is not necessary any package for the build process.

**Comment by williamjamir** [comment]
It is not necessary any extra package, you are correct. I was basing my recipe from the output of conda skeleton, but looking at setup.py I noticed that is not necessary any package for the build process.

**Comment by synapticarbors** [comment]
@williamjamir can you just switch all of the build numbers back to `0` and then I think it should be ready to merge.

**Comment by synapticarbors** [comment]
Also, if you'd submit issues upstream about including the license file in the source distribution and copy the links as comments into each corresponding recipe as a reminder to look for future updates, that would be great.

**Comment by williamjamir** [comment]
@synapticarbors , this last commit increment the version of the packages, and with this new versions, the LICENSE file is included in the source distribution =)

About the CI, the appveyor looks good (opening the website the build is green) but the travis-ci didn't start yet. Should I do something related to this?

**Comment by synapticarbors** [comment]
@williamjamir everything looks good to me now and Travis has started, so once everything runs, I'll merge. Just be warned that there has been a consistent Travis backlog of >24 hr so it may take some time.

**Comment by williamjamir** [comment]
@synapticarbors I think that is ready to merge =)
(appveyor is buggy, opening the website the build passed but the icon didn't change the color)

**Pull request merged**